src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -286,12 +295,30 @@ private BigQueryWriter getWriter() {
     long retryWait = config.getLong(config.BIGQUERY_RETRY_WAIT_CONFIG);
     BigQuery bigQuery = getBigQuery();
     if (updateSchemas) {
-      return new AdaptiveBigQueryWriter(bigQuery, getSchemaManager(bigQuery), retry, retryWait);
+      return new AdaptiveBigQueryWriter(bigQuery, getSchemaManager(bigQuery), retry, retryWait,  metrics);
     } else {
-      return new SimpleBigQueryWriter(bigQuery, retry, retryWait);
+      return new SimpleBigQueryWriter(bigQuery, retry, retryWait, metrics);
     }
   }
 
+  private void configureMetrics() {
+    // todo more metric configuration?
+    metrics = new Metrics();
+    rowsRead = metrics.sensor("rows-read");
+    rowsRead.add(metrics.metricName("rows-written-avg",
+                                    "kafka", // I have no idea how the group name is used or what it should be...
```

Comment:
Hmm, I could be confused about what group name means. I'll defer to whatever pattern Kafka is using.